The default spell checker sends everything you type to some remote server
(http://spell.roundcube.net). This is de facto keylogger!

This patch just changes the default settings to disable spellcheck
and set pspell as default (however it requires php7-pspell to be installed).

--- a/config/defaults.inc.php
+++ b/config/defaults.inc.php
@@ -723,7 +723,7 @@
 $config['quota_zero_as_unlimited'] = false;
 
 // Make use of the built-in spell checker. It is based on GoogieSpell.
-$config['enable_spellcheck'] = true;
+$config['enable_spellcheck'] = false;
 
 // Enables spellchecker exceptions dictionary.
 // Setting it to 'shared' will make the dictionary shared by all users.
@@ -737,7 +737,7 @@
 // Since Google shut down their public spell checking service, the default settings
 // connect to http://spell.roundcube.net which is a hosted service provided by Roundcube.
 // You can connect to any other googie-compliant service by setting 'spellcheck_uri' accordingly.
-$config['spellcheck_engine'] = 'googie';
+$config['spellcheck_engine'] = 'pspell';
 
 // For locally installed Nox Spell Server or After the Deadline services,
 // please specify the URI to call it.
